docs/webassembly-wasmrs-iotas-rust.md

```diff
@@ -0,0 +1,117 @@
+# Extending services with WebAssembly
+
+Nanobus is a way to stitch together logic, no matter where it exists or what form it's in. We call bundles of such logic "iotas" meaning "small things". Iotas are simply a collection of interfaces that expose actions (i.e. functions). They're just like language libraries or modules you've used before, except with an interface that fits distributed, cross-language, cross-platform usage.
+
+Iotas need to communicate with known protocols and WebAssembly iotas use RSocket via wasmRS to establish bidirectional communication between the WebAssembly guest and the iota host, nanobus.
+
+## Starting off
+
+Use [apex] to start a new project for your language of choice. This command uses the template found in the path `templates/rust` from the repository `git@github.com:nanobus/iota.git`. Any git repository can be an apex template.
+
+Use the project name `jinja` for this example. We'll be building a template-rendering iota.
+
+```
+apex new git@github.com:nanobus/iota.git -p templates/rust jinja
```

Comment:
I can't figure out what this needs to be.  I installed Apex using `deno install -A --unstable -f -n apex https://deno.land/x/apex_cli/apex.ts` and don't have any -p command.